services/movie.py

```diff
@@ -22,12 +28,14 @@ def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
 
 
+@transaction.atomic
 def create_movie(
     movie_title: str,
     movie_description: str,
     genres_ids: list = None,
     actors_ids: list = None,
-) -> Movie:
+) -> QuerySet:
```

Comment:
The return type for `create_movie` is incorrectly specified as `QuerySet`. Since this function returns a single `Movie` instance, the return type should be `Movie` instead.